### ISSUE TITLE
Error "Cannot find module 'i18n:astro' or its corresponding type declarations." on a fresh Astro project

### ISSUE THREAD

**Issue by quasiDigi** [florian-lefebvre/astro-i18n-test#32]
### What I did:
1. Created a new Astro project through npm, options > ts: strictest, template: blog, install dependencies
```
npm create astro@latest
```

2. I've added  [I18n for Astro](https://astro-i18n.netlify.app/) > options: yes to all
```
npm run astro add @astrolicious/i18n
```
3. Added `routes` and `locales` folders in `src`, if not dev server through `npx astro dev` wouldn't launch

4. On my BlogPost layout I try to get the locale, as explained in the [docs](https://astro-i18n.netlify.app/getting-started/usage/):
```js
---
import { getLocale } from "i18n:astro"

const locale = getLocale()
---
```

### Result:
I get the following error:
```
Cannot find module 'i18n:astro' or its corresponding type declarations.
```

best,
q.

**Comment by florian-lefebvre** [comment]
Hi! It needs to write types to your project, you need to run `npm run astro sync`. Let me know if that works!

**Comment by quasiDigi** [comment]
Hi, thanks for the fast reply! 
I forgot to mention that I did the npx version `npx astro sync` yesterday. Today, I did `npm run astro sync` and the error message is still there. Nevertheless, it seems to work, as `console.log("locale: ", locale);` after the `const locale = getLocale();` is outputting the locale.

**Comment by florian-lefebvre** [comment]
Ah I think I know why. It's most likely an upstream issue, I'll push a fix this afternoon https://github.com/florian-lefebvre/astro-integration-kit/issues/117

**Comment by florian-lefebvre** [comment]
Fix should be available in `0.5.0` (note it requires Astro >= 4.12.0)
